File: sdkwork-app-sdk-python/sdkwork_app_sdk/api/sms.py
from typing import Any, Dict, List, Optional
from ..http_client import HttpClient
from ..models import PlusApiResultVerifyResultVO, PlusApiResultVoid, VerifyCodeCheckForm, VerifyCodeSendForm

class SmsApi:
    """sms API client."""
    
    def __init__(self, client: HttpClient):
        self._client = client

    def verify_sms_code(self, body: VerifyCodeCheckForm) -> PlusApiResultVerifyResultVO:
        """验证验证码"""
        return self._client.post(f"/app/v3/api/auth/sms/verify", json=body)

    def send_sms_code(self, body: VerifyCodeSendForm) -> PlusApiResultVoid:
        """发送验证码"""
        return self._client.post(f"/app/v3/api/auth/sms/send", json=body)

File: sdkwork-app-sdk-flutter/lib/src/api/sms.dart
import '../http/client.dart';
import '../models.dart';

class SmsApi {
  final HttpClient _client;
  
  SmsApi(this._client);

  /// 验证验证码
  Future<PlusApiResultVerifyResultVO?> verifySmsCode(VerifyCodeCheckForm body) async {
    final response = await _client.post(ApiPaths.appPath('/auth/sms/verify'), body: body, contentType: 'application/json');
    return response is PlusApiResultVerifyResultVO ? response : null;
  }

  /// 发送验证码
  Future<PlusApiResultVoid?> sendSmsCode(VerifyCodeSendForm body) async {
    final response = await _client.post(ApiPaths.appPath('/auth/sms/send'), body: body, contentType: 'application/json');
    return response is PlusApiResultVoid ? response : null;
  }
}
